### ISSUE TITLE
gf doesn't work for partials for me?

### ISSUE THREAD

**Issue by thomaswitt** [tpope/vim-rails#602]
Hi everybody,

I have a strange thing, gf works fine for me e.g. when in a Controller, it jumps to the model, when I have a partial, it jumps to the partial.

But when I render something from layout, it doesnt work: `<%= render 'layouts/search' %>`

gf on layouts/search doesn't work. All I get is E345: Can't fnd file "layouts/search" in path …

 Yes, I have seen [other](https://github.com/tpope/vim-rails/issues/347) [issues](https://github.com/tpope/vim-rails/pull/149) and therefore it should be working but I don't know why it doesn't. Any idea?

**Comment by thewatts** [comment]
Mine is doing this too, after switching to using LazyVim and updating my config to use lua.

:) I'm still working on figuring it out - but I'll update this here if I have any luck

**Comment by thewatts** [comment]
Update: It looks like `nvim-treesitter` is causing things to break on my end.

If I disable it, `gf` starts working again.

**Comment by awortham** [comment]
Agreed -> nvim-treesitter seems to break the `gf` command to go find partials. Is there a way to give vim-rails command precedence? I'm assuming something in tree-sitter must also be relying on that same command.

**Comment by thewatts** [comment]
Potentially related: https://github.com/nvim-treesitter/nvim-treesitter/issues/6187

**Comment by natematykiewicz** [comment]
My `gf` couldn't find partials either. After adding `additional_vim_regex_highlighting = true,` to my treesitter config per [this comment](https://github.com/nvim-treesitter/nvim-treesitter/issues/6187#issuecomment-1962257273), `gf` works as expected.

**Comment by jiz4oh** [comment]
or add the following code in your .vimrc if you don't want set `additional_vim_regex_highlighting = true`

```
augroup vim-rails-augroup
  autocmd FileType eruby
        \ if RailsDetect() | call rails#ruby_setup() | endif

augroup END
```
